### PR TITLE
Added picture grid directive

### PR DESCRIPTION
**Pull request by DanielBrierton**

### CODE REVIEW

**Comment by butlerx** [comment]
LGTM

File: web/public/js/directives/cd-picture-grid/template.dust [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<div class="cd-picture-grid">
+    <a class="cd-picture-grid__item" ng-repeat="item in $ctrl.items" href="{{ item.href }}">
```
Comment (Wardormeur) [inline web/public/js/directives/cd-picture-grid/template.dust]: should'nt it be ng-href?

File: web/public/js/directives/cd-picture-grid/template.dust [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+        <div class="cd-picture-grid__picture" style="background-image: url({{ item.picture }});"></div>
```
Comment (Wardormeur) [inline web/public/js/directives/cd-picture-grid/template.dust]: same here, ng-style?

File: web/public/js/directives/cd-picture-grid/template.dust (hunk 1 above)
Comment (DanielBrierton) [inline web/public/js/directives/cd-picture-grid/template.dust]: Fair enough, changed that now, will push in a min

File: web/public/js/directives/cd-picture-grid/template.dust (hunk 2 above)
Comment (DanielBrierton) [inline web/public/js/directives/cd-picture-grid/template.dust]: ng-style is for applying styles from objects in the scope. The items array would then have to be:
```
[{
  href: '/whatever',
  caption: 'Whatever',
  picture: {
    'background-image': 'url(/whatever.png)'
  }
}]
```

Seems easier for users of the directive to leave it as is.

Comment (Wardormeur) [inline web/public/js/directives/cd-picture-grid/template.dust]: Fair, forgot ng-style was oriented on optional styling (the same as ng-class)

**Pull request merged**